### PR TITLE
Adds tasks.json to easily use VSCode with TypeScript sources

### PR DESCRIPTION
**Pull request by rbuckton**

### CODE REVIEW

File: .settings/tasks.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Available variables which can be used inside of strings.
+// ${workspaceRoot}: the root folder of the team
+// ${file}: the current opened file
+// ${fileBasename}: the current opened file's basename
+// ${fileDirname}: the current opened file's dirname
+// ${fileExtname}: the current opened file's extension
+// ${cwd}: the current working directory of the spawned process
```
Comment (DanielRosenwasser) [inline .settings/tasks.json]: Is it only VSCode that uses this `tasks.json`? I know that VSCode permits comments in json, but does anything else consume this?

Comment (rbuckton) [inline .settings/tasks.json]: I think it is only VSCode that uses this.

**Comment by vladima** [comment]
:+1:

**Comment by DanielRosenwasser** [comment]
Can we make this use 4 spaces to be consistent with the rest of our `json` files?

**Pull request merged**